requirements.txt

```diff
@@ -0,0 +1,5 @@
+flake8
+coveralls
+Sphinx
+Django>1.7,<1.9
```

Comment:
Whoops, just noticed this `<1.9` which will stop 1.9.x from being installed! I'll fix it. :smile: